### PR TITLE
added support for back drop press

### PR DESCRIPTION
**Pull request by injas-bigbinary**
@Tsudhishnair _a

### CODE REVIEW

File: src/components/MultiSelect.js [hunk 1]
```diff
@@ -623,6 +625,10 @@ MultiSelect.propTypes = {
    * Custom component to render more item message
    */
   MoreItemComponent: PropTypes.node,
+  /**
+   * FUnction to customize back drop press
```
Comment (Tsudhishnair) [inline src/components/MultiSelect.js:629]: `FUnction` -> `Function`

File: src/components/BottomSheet.js [hunk 2]
```diff
@@ -409,6 +414,10 @@ BottomSheet.propTypes = {
    * Use custom key as value.
    */
   valueExtractor: PropTypes.func,
+  /**
+   * FUnction to customize back drop press
```
Comment (Tsudhishnair) [inline src/components/BottomSheet.js:418]: `FUnction` -> `Function`

File: storybook/stories/MultiSelect.stories.js [hunk 3]
```diff
@@ -73,6 +73,9 @@ export const MultiSelects = () => {
       )}
       valueExtractor={item => item.id}
       labelExtractor={item => item.name}
+      // onBackdropPress={() => {
+      //   console.log("ok");
```
Comment (Tsudhishnair) [inline storybook/stories/MultiSelect.stories.js:77]: We can just make this as an empty function. `()=>{}`

**Review by Tsudhishnair — CHANGES_REQUESTED** [review]
Code LGTM. 
You can merge after fixing the conflict and the other mentioned minor changes.

**Comment by Tsudhishnair** [comment]
@injas-bigbinary _a

**Comment by injas-bigbinary** [comment]
@Tsudhishnair _a

**Comment by Tsudhishnair** [comment]
@injas-bigbinary I will push the multiSelect fix in this PR which we discussed over the call.

**Pull request merged**